src/context/Localization/language/en.json

```diff
@@ -1148,7 +1148,25 @@
     "SettingUpTapsigner": "Setting up TAPSIGNER",
     "AlreadyInitialized": "Already Initialized",
     "Uninitialized": "Uninitialized",
-    "NotBeenInitializedBottomText": "This card has not been initialized before. You may proceed with the setup."
+    "NotBeenInitializedBottomText": "This card has not been initialized before. You may proceed with the setup.",
+    "addKeyHardware": "Add key from a hardware",
+    "connectHardware": "Connect your hardware device",
+    "hardwareKeysHeader": "Hardware Keys",
+    "connectHardwareDevices": "Connect your Hardware devices",
+    "addSoftwareKey": "Add a software key",
+    "keysInApp": "Keys generated within Keeper App",
```

Comment:
This isn't exactly accurate since it can also be seed imported from elsewhere or another software signer. @ASN-BitHyve we need better phrasing for this

Reply:
"In-app or external software signers"